.github/workflows/issue-tests.yml

```diff
@@ -29,6 +29,14 @@ jobs:
 
     - name: Setup YugabyteDB
       run: |
+        # Note: Update val for LATEST_STABLE here along with 'src/ybversion/constants.go/LATEST_STABLE' variable
+        if [ "${YB_VERSION}" == "LATEST_STABLE" ]; then
+          YB_VERSION="2024.2.2.0-b61"
+        fi
+
+        # write it to GitHub's environment to override previous value
```

Comment:
don't we want to do this only in the if-block above?

Reply:
yeah correct, although end result would be same but inside if block is correct place.

Reply:
done